js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,7 @@
+// import { isString } from "util";
+
+function abbreviateString(str) {
+  return `${str.slice(0, str.indexOf(' '))} ${str.charAt(str.lastIndexOf(' ') + 1).toUpperCase()}.`;
```

Comment:
@Abhijeet-pesto Please do not try to do too many things in a single statement. It makes your code very difficult to read.